### ISSUE TITLE
v1.1.3 unable to pair

### ISSUE THREAD

**Issue by matthewyang204** [bsharper/atv-desktop-remote#12]
Unable to pair v1.1.3 with any Apple TV. v1.1.1 works fine.

**Comment by yetelday** [comment]
Same issue here, keeps scanning while prompting for the 4 digit code that never appears on the appleTV.

**Comment by gitstua** [comment]
I was able to install and pair with an older release https://github.com/bsharper/atv-desktop-remote/releases/tag/v1.0.0-beta

**Comment by cgaff91** [comment]
Same issue with 1.1.3. Looks like the 1.1.1 build works fine. Great app!

**Comment by matthewyang204** [comment]
Yeah, I'm on v1.1.1 and I'm fine, too. Just that v1.1.3 gets fixed eventually. bsharper should mark that release as pre-release.

**Comment by robertalanharris** [comment]
Also having issues pairing with v1.1.3-arm46.  v1.1.1 works just fine.

**Comment by wesgurn** [comment]
Confirming that v1.1.1-arm64 pairing works with AppleTV 4k.  Couldn't pair it with v1.1.3.

**Comment by bsharper** [comment]
Thanks for the feedback everyone. I've reverted 1.1.3 to a pre-release so 1.1.1 is the latest version, which should prevent people from seeing this bug right away. I'm looking into what caused the bug.

**Comment by matthewyang204** [comment]
Okay

**Comment by dmrowe** [comment]
I had the same problem as others have described here. Installing 1.1.1 allowed me to pair; I then installed 1.1.3 over the top and it works as expected. Just adding FYI.
